pkg/builder/local_build_executor.go

```diff
@@ -248,6 +253,24 @@ func (be *localBuildExecutor) Execute(ctx context.Context, filePool re_filesyste
 		return response
 	}
 
+	if len(be.inputRootCharacterDevices) > 0 {
+		if err := inputRootDirectory.Mkdir("dev", 0777); err != nil && !os.IsExist(err) {
+			attachErrorToExecuteResponse(response, util.StatusWrap(err, "Unable to create dev directory in input root"))
+		}
+		devDir, err := inputRootDirectory.EnterDirectory("dev")
+		if err != nil {
+			attachErrorToExecuteResponse(response, util.StatusWrap(err, "Unable to enter dev directory in input root"))
+		}
+		for name, number := range be.inputRootCharacterDevices {
+			if _, err := devDir.Lstat(name); err == nil {
+				continue
+			}
+			if err := devDir.Mknod(name, os.FileMode(syscall.S_IFCHR|0666), number); err != nil {
```

Comment:
This should likely use `os.ModeDevice | os.ModeCharDevice | 0666`.

Reply:
That creates a different value which I'm fairly sure doesn't create a device node when used with Mknod as I tried it before:
```
2020/05/28 10:24:16 mode|os.ModeDevice|os.ModeCharDevice: 0x42001b6
2020/05/28 10:24:16 mode|syscall.S_IFCHR: 0x21b6
```

Reply:
Sure, but what I’m getting at is that S_IFCHR is not a value that you’re normally supposed to store in an os.FileMode. We either use os.FileMode in combination with the intended numerical constants, or we don’t.

Reply:
So it may be best to not have Mknod take `os.FileMode` as and argument and just let it be `uint32` directly? Then I could just not cast to `os.FileMode` and pass `syscall.S_IFCHR|0666` directly?

Reply:
Though that is one option to do it, I would suggest we stick to using os.FileMode and let this code pass in the os.Mode* constants. That avoids a dependency on sys/unix in this code. Having such a dependency may be problematic if we ever want to make Windows support more complete.

Reply:
Or pass the `os.Mode...` values as you suggested and handle it on the other end